servers/audio/effects/audio_effect_pitch_shift.cpp

```diff
@@ -92,7 +92,7 @@ void SMBPitchShift::PitchShift(float pitchShift, long numSampsToProcess, long ff
 	fftFrameSize2 = fftFrameSize/2;
 	stepSize = fftFrameSize/osamp;
 	freqPerBin = sampleRate/(double)fftFrameSize;
-	expct = 2.*Math_PI*(double)stepSize/(double)fftFrameSize;
+	expct = Math_TAU*(double)stepSize/(double)fftFrameSize;
```

Comment:
I know that there's a comment above which says `/* Thirdparty code,`, but this part of the code has already been modified from the source anyway (`M_PI` -> `Math_PI`) so I think this change is good.

Reply:
Performance wise this doesn't make any difference, the compiler optimize all these constants at compile time. So there is no much benefit modifying a third party source: on the next update such change will be lost.